birdspotter/birdspotter/dataviz/views.py

```diff
@@ -1,7 +1,55 @@
 from django.shortcuts import render
 from django.http import HttpResponse
+import plotly
+import plotly.express as px
+import plotly.graph_objects as go
+import random
+import pandas as pd
 
 # Create your views here.
-
+# Data view
 def index(request):
-	return HttpResponse("This is the data index")
\ No newline at end of file
+    args = {}
+
+    # Total bird count data
+    birds = ["Gull", "Razorbill", "Common Eider", "Sandpiper"
+             , "Cormorants", "Great Cormorant"]
+    data_total = {'Specie': [birds[i] for i in range(len(birds))],
```

Comment:
Can this just be `'Specie': birds`?

Reply:
It creates an error. "arrays must all be same length"

Reply:
Okay, I fixed it. Should look better now.